plugins/woocommerce-admin/package.json

```diff
@@ -226,7 +226,6 @@
 		]
 	},
 	"engines": {
-		"node": "^16.13.1",
-		"pnpm": "^6.24.2"
+		"node": "^16.13.1"
```

Comment:
I've opted to remove this rather than set it to a later version. I think we should only fix if we see issues and need to lock it. We could also consider [corepack](https://github.com/nodejs/corepack) at that point.